launchers/configure.sh

```diff
@@ -75,7 +75,7 @@ if [ "x$ITW_LIBS" == "xDISTRIBUTION" ] ; then
   readonly JAVAWS_SRC=`ls $PROJECT_TOP/artifact-no-dependencies/target/icedtea-web-no-dependencies-*.jar | grep -v sources | grep -v javadoc`
   readonly JAVAWS_SRC_SRC=`ls $PROJECT_TOP/artifact-no-dependencies/target/icedtea-web-no-dependencies-*.jar | grep sources`
 else
-  readonly JAVAWS_SRC=`ls $PROJECT_TOP/artifact-all-dependencies/target/icedtea-web-all-dependencies-*.jar | grep -v sources | grep -v javadoc`
+  readonly JAVAWS_SRC=`ls $PROJECT_TOP/artifact-all-dependencies/target/icedtea-web-all-dependencies-*-shaded.jar`
```

Comment:
If I check in my target folder maven is currently building no `icedtea-web-all-dependencies-*-shaded.jar`.
Do you see such a file in your target folder??

Reply:
Yes, with these command lines:

```
mvn clean install -DskipTests=true
ls -l artifact-all-dependencies/target/
```
 => no shaded jar

```
mvn install -DskipTests=true -Plaunchers
ls -l artifact-all-dependencies/target/
```
 => shaded jar

```
$ ls -l artifact-all-dependencies/target/
total 11496
-rwxrwx---+ 1 2678467 31 oct.  23:34 icedtea-web-all-dependencies-2.0.0-SNAPSHOT.jar
-rwxrwx---+ 1 2678467 31 oct.  23:34 icedtea-web-all-dependencies-2.0.0-SNAPSHOT-shaded.jar
-rwxrwx---+ 1 1866323 31 oct.  23:34 icedtea-web-all-dependencies-2.0.0-SNAPSHOT-sources.jar
drwxrwx---+ 1       0 31 oct.  23:30 maven-archiver
-rwxrwx---+ 1 2678466 31 oct.  23:34 original-icedtea-web-all-dependencies-2.0.0-SNAPSHOT.jar
-rwxrwx---+ 1 1866322 31 oct.  23:30 original-icedtea-web-all-dependencies-2.0.0-SNAPSHOT-sources.jar

```

Reply:
The "-shaded" jar is only a temporary file created by the maven-shade-plugin.
Once the plugin finishes it will replace the original jar with the temporary "-shaded" jar.

So you should not rely on this shaded file as it will only exist during the execution of the maven-shade-plugin

See https://github.com/apache/maven-shade-plugin/blob/master/src/main/java/org/apache/maven/plugins/shade/mojo/ShadeMojo.java#L514

Reply:
OK I have updated the PR to exclude shaded jar